kiva/fonttools/tests/test_font_manager.py

```diff
@@ -93,3 +109,162 @@ def test_font_with_italic_style(self):
         self.assertEqual(entry.weight, exp_weight)
         self.assertEqual(entry.stretch, exp_stretch)
         self.assertEqual(entry.size, exp_size)
+
+
+class TestFontCache(unittest.TestCase):
+    """ Test internal font cache building mechanism."""
+
+    def setUp(self):
+        self.ttf_files = [
+            os.path.abspath(os.path.join(data_dir, "TestTTF.ttf"))
+        ]
+
+        self.temp_dir = tempfile.mkdtemp()
+        self.addCleanup(shutil.rmtree, self.temp_dir)
```

Comment:
What do you think about using `tempfile.TemporaryDirectory` and its `cleanup` method? ([docs](https://docs.python.org/3.6/library/tempfile.html#tempfile.TemporaryDirectory))